lib/code_climate/test_reporter/git.rb

```diff
@@ -17,16 +16,16 @@ def branch_from_git_or_ci
 
         def clean_service_branch
           ci_branch = String(Ci.service_data[:branch])
-          clean = ci_branch.strip.sub(/^origin\//, "")
+          clean = ci_branch.strip.sub(%r{^origin\/}, "")
 
-          clean.size > 0 ? clean : nil
+          !clean.empty? ? clean : nil
         end
 
         def clean_git_branch
           git_branch = String(branch_from_git)
-          clean = git_branch.sub(/^origin\//, "") unless git_branch.start_with?("(")
+          clean = git_branch.sub(%r{^origin\/}, "") unless git_branch.start_with?("(")
```

Comment:
(This, apparently, is called ["leaning toothpick syndrome"](https://en.wikipedia.org/wiki/Leaning_toothpick_syndrome))